Probesys/bileto
# New/about

Related to https://github.com/Probesys/bileto/issues/85

Changes proposed in this pull request:

- Improve the look of the popups
- Rename the HomeController into PagesController
- Create a "about" action to render the about page
- Add links to the about page from the main menu and the login page

How to test the feature manually:

1. Open Bileto disconnected, click on "About Bileto"
2. Verify the content looks good
3. Login and open the main menu, click on About Bileto
4. Verify you can access the about page

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
